package.json

```diff
@@ -79,10 +80,13 @@
       "last 1 safari version"
     ]
   },
+  "resolutions": {
+    "react-error-overlay": "6.0.9"
```

Comment:
Why we need it here?

Reply:
Couse I have [this bug](https://github.com/facebook/create-react-app/issues/11773) and it helped me to fix it.